test/attr/lexical.swift

```diff
@@ -44,13 +44,13 @@ func foo() {
   _ = s3
 }
 
-@_moveOnly struct MoveOnly {}
+struct MoveOnly: ~Copyable {}
 
-@_eagerMove @_moveOnly struct MoveOnlyEagerly {} // expected-error {{@_eagerMove cannot be applied to NonCopyable types}}
+@_eagerMove struct MoveOnlyEagerly: ~Copyable {} // expected-error {{@_eagerMove cannot be applied to NonCopyable types}}
 
 func zoo(@_eagerMove _ : consuming MoveOnly) {} // expected-error {{@_eagerMove cannot be applied to NonCopyable types}}
 
-func zooo(@_noEagerMove  _ : consuming C) {} // ok, only way to spell this behavior
+func zooo(@_noEagerMove  _ : C) {} // ok, only way to spell this behavior
```

Comment:
@nate-chandler does this still test what it's meant to?

Reply:
No, it's supposed to verify that it's legal to decorate a Copyable argument with both `@_noEagerMove` and `consuming`.  Since this PR bans-for-now consuming Copyable arguments, that case doesn't exist anymore.  Ideally this test would be commented out with a TODO, but this is fine.

Reply:
I went with the TODO!